### DISCUSSION TITLE
Allow users to specify zoom range?

### DISCUSSION THREAD

**Discussion by omar-shahid** [xyflow/xyflow#2321] [New Features]
Hi React Flow Renderer Team!
Thank you so much for creating this awesome library. You guys are awesome.

I was wondering whether we can have custom zoom range so developer can specify how small or large zoom will be allowed for user. I searched the documentation but got no luck.

I saw that the React Flow is using CSS transforms "scale for zooming in and out " for panning and zoom. Still not sure what would be the edge cases with this feature.

Many Thanks.

**Comment by moklick** [comment]
Thanks @theJsTsGuy !

You can use [`minZoom` and `maxZoom`](https://reactflow.dev/docs/api/react-flow-props/#flow-view) to define the zoom range.

**Comment by omar-shahid** [comment]
Thanks for the quick response @moklick ! Highly appreciate it. Thanks for making this amazing product once again!
